packages/flutter_web_plugins/lib/src/plugin_registry.dart

```diff
@@ -195,6 +195,10 @@ class PluginRegistry extends Registrar {
   /// Creates a [Registrar].
   ///
   /// The argument is ignored.
+  @Deprecated(
+    'Use Registrar instead. '
+    'This feature was deprecated after v1.24.0-7.0.pre.'
```

Comment:
@a14n I think here should `v1.26.0-18.0.pre`.

Reply:
I used the same deprecation notice as the one on the class. They should be consistent.

Reply:
Thanks for your explanation.

Reply:
Since this is not being removed soon (like typematcher), we should reset the deprecation clock like we did for those that were missing in https://github.com/flutter/flutter/pull/75813.
Typematcher has already been announced on multiple channels over the last few months as being on the way out so folks are aware. :)